docs/userguide/transfer_learning.md

```diff
@@ -1,4 +1,41 @@
 # Transfer Learning
 
-This page will soon contain information about transfer learning.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+BayBE offers the possibility to mix data from multiple campaigns in order to accelerate
+optimization. 
+Using data from multiple campaigns is currently supported when using a Gaussian Process
+Surrogate model and is implemented by the [`TaskParameter`](baybe.parameters.categorical.TaskParameter).
+
+```{admonition} Terminology
+:class: note
+The term "Transfer Learning" is used in a lot of different ways.
```

Comment:
youve started to describe here the different ways of inclding transfer learning, hwoever when staring with saying something about terminology i was expecting tht this decribes that "transfer learning" is ur way of saying "combining data of multiple capaign" and theres alternatives such as "contextual learning" or somethign else I dont remember rght now

Reply:
Changed the formulation a bit, is this now better?

Reply:
yep